jerry-debugger/src/lib/cdt-proxy.ts

```diff
@@ -36,13 +44,62 @@ export class ChromeDevToolsProxyServer {
       ws.send('hello');
     });
 
+    server.listen(SERVER_PORT);
+
+    // TODO: move callback out to standalone method
+    server.on('request', function (request: http.IncomingMessage, response: http.ServerResponse) {
+      var url = request.url || '';
+      if (url == '/json') {
+        url = '/json/list';
+      }
+
+      if (request.method != "GET" || url.slice(0, 6) != '/json/') {
```

Comment:
Please always use `!==` and `===` (strict (in)equal).
The linter (`yarn lint`) should also point it out.
The downside of `!=` and `==` operators is that they silently coerce/convert the types if they don't match, which can result in very strange bugs.